static-assets/yui/container/container.js

```diff
@@ -5743,10 +5743,10 @@ version: 2.6.0
          * @param {Event} e The DOM event object
          */
         _onElementFocus : function(e){
-
+            var authDialog = document.getElementById('auth-monitor-dialog');
```

Comment:
Camelcase ID.
Missing the AuthMonitor.tsx update that adds this ID to the dialog.

Please revert all other changes in this file, leaving only the code changes. Make sure not to format. If necessary, edit the file with something other than IntelliJ so it doesn't remove trailing whitespaces and things of that nature.